app/src/pages/playground/schemas.ts

```diff
@@ -136,8 +136,8 @@ const stringToInvocationParametersSchema = z
   .string()
   .transform((s) => {
     const { json } = safelyParseJSON(s);
-    if (json == null) {
-      return {};
+    if (json == null || typeof json !== "object") {
```

Comment:
i think there might be a isObject in src/typeUtils, don't hafve to use it here though